src/UI-Constants/Constant.json

```diff
@@ -133,5 +136,47 @@
     "TIMEOUT_MEDIUM": 10000,
     "TIMEOUT_LONG": 30000,
     "TIMEOUT_XLONG": 60000
+  },
+  "Styles": {
```

Comment:
Why is styles here ? move it to some other file

Reply:
Moved styles to styles.tsx